docs/tempo/website/configuration/_index.md

```diff
@@ -10,15 +10,14 @@ This section explains the configuration options for Tempo as well as the details
   - [Authentication/Server](#authenticationserver)
   - [Distributor](#distributor)
   - [Ingester](#ingester)
-  - [Query Frontend](#queryfrontend)
+  - [Query Frontend](#query-frontend)
   - [Querier](#querier)
   - [Compactor](#compactor)
   - [Storage](#storage)
   - [Memberlist](#memberlist)
-  - [Compression](#compression)
 
 ## Authentication/Server
-Tempo uses the Weaveworks/common server.  See [here](https://github.com/weaveworks/common/blob/main/server/server.go#L45) for all configuration options.
+Tempo uses the Weaveworks/common server. See [here](https://github.com/weaveworks/common/blob/master/server/server.go#L45) for all configuration options.
```

Comment:
I'm not sure why this had to change from `main` to `master`, but I can't find a `main` branch on https://github.com/weaveworks/common/branches 🤷🏻

Reply:
Probably `s/master/main/` (on #655) changed this, and no-one noticed :smile:

Reply:
Ah, makes sense. Another mystery solved 😃